CapabilityStatements/CapabilityStatement-mp-MedicationData.RetrieveServe.Base.xml

```diff
@@ -102,6 +77,18 @@
         <definition value="http://nictiz.nl/fhir/SearchParameter/next-practitioner"/>
         <type value="reference"/>
       </searchParam>
+      <searchParam>
+        <name value="dispense-location"/>
+        <definition value="http://nictiz.nl/fhir/SearchParameter/dispense-location"/>
+        <type value="reference"/>
+        <documentation value='This search parameter only needs to be supported for MedicationRequest resources that conform to the mp-DispenseRequest profiles.  In these cases the _include of the dispense-location needs to be supported as well.'/>
+      </searchParam>
+      <searchParam>
+        <name value="reason"/>
+        <definition value="http://nictiz.nl/fhir/SearchParameter/reason"/>
+        <type value="reference"/>
+        <documentation value='This search parameter only needs to be supported for MedicationRequest resources that conform to the mp-MedicationAgreement and Mp-VariableDosingRegimen profiles.'/>
```

Comment:
Replace 'Mp-' by 'mp-', and add the sentence 'In these cases the _include of the reason needs to be supported as well.' at the end (i.e. copy line 362 here).